revert drm/atomic: Use drm_drv_uses_atomic_modeset() for debugfs creation

3c499ea0c662e2f38aafbd4f516b08aab8cfa0e1 introduces an oops on radeon
graphics, revert it from v4.18.

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>

---
 drivers/gpu/drm/drm_atomic.c  |    2 +-
 drivers/gpu/drm/drm_debugfs.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/gpu/drm/drm_atomic.c
+++ b/drivers/gpu/drm/drm_atomic.c
@@ -1838,7 +1838,7 @@ static void __drm_state_dump(struct drm_
 	struct drm_connector *connector;
 	struct drm_connector_list_iter conn_iter;
 
-	if (!drm_drv_uses_atomic_modeset(dev))
+	if (!drm_core_check_feature(dev, DRIVER_ATOMIC))
 		return;
 
 	list_for_each_entry(plane, &config->plane_list, head) {
--- a/drivers/gpu/drm/drm_debugfs.c
+++ b/drivers/gpu/drm/drm_debugfs.c
@@ -150,7 +150,7 @@ int drm_debugfs_init(struct drm_minor *m
 		return ret;
 	}
 
-	if (drm_drv_uses_atomic_modeset(dev)) {
+	if (drm_core_check_feature(dev, DRIVER_ATOMIC)) {
 		ret = drm_atomic_debugfs_init(minor);
 		if (ret) {
 			DRM_ERROR("Failed to create atomic debugfs files\n");
